src/molecules/side-menu.tsx

```diff
@@ -0,0 +1,26 @@
+import styles from './side-menu.module.css';
+
+const SideMenu = ({ isOpen, closeMenu, subjects }) => {
+  return (
+    <>
+      {isOpen && <div className={styles.overlay} onClick={closeMenu}></div>}
+      <div className={styles.sideMenu} data-open={isOpen ? 'true' : 'false'}>
+        <div className={styles.topBar}>
+          <button className={styles.closeButton} onClick={closeMenu}>×</button>
```

Comment:
```suggestion
          <span className="title-small">出席確認システム</span>
          <button className={styles.closeButton} onClick={closeMenu}>×</button>
```
この閉じるボタンの左側に, このような具合で「出席確認システム」というアプリ名のラベルを入れてほしいです.